src/MovieLogger.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import FlixterHeader from "./FlixterHeader";
+import "./MovieLogger.css";
+function MovieLogger() {
+  const [movies, setMovies] = useState([]);
+  const [currentMovie, setCurrentMovie] = useState({ name: "", rating: "" });
+  const [comparisonIndex, setComparisonIndex] = useState(0);
+  const [isComparing, setIsComparing] = useState(false);
+
+  const handleMovieInput = (e) => {
+    setCurrentMovie({ ...currentMovie, name: e.target.value });
+  };
+
+  const handleRatingSelect = (e) => {
+    setCurrentMovie({ ...currentMovie, rating: e.target.value });
+  };
+
+  const startComparison = () => {
+    if (currentMovie.name && currentMovie.rating) {
+      setIsComparing(true);
+      setComparisonIndex(0);
+    } else {
+      alert("Please enter movie details and select a rating.");
```

Comment:
As an addition, consider using one of the free react notification libraries

Reply:
@samyalihamad I think this can be planned as a stretch goal for the project.

Reply:
Sounds good I can add it to the project plan